base/src/main/java/org/aya/ref/LocalVar.java

```diff
@@ -7,10 +7,17 @@
 import org.jetbrains.annotations.NotNull;
 import org.jetbrains.annotations.Nullable;
 
+import java.util.UUID;
+
 /**
  * @author ice1000
  */
 public record LocalVar(@NotNull String name, @NotNull SourcePos definition) implements Var {
+
+  public static LocalVar IGNORED_LOCAL() {
+    return new LocalVar("AYA_INTERNAL_IGNORED_LOCALVAR" + UUID.randomUUID());
```

Comment:
If it's ignored, is it never substituted? Can we make it a constant instead?

Reply:
I'm not sure non-unique names in lambda arguments are acceptable, if so then constants probably wouldn't work.

Reply:
Will address this in a future PR

Reply:
This is a special name anyway.